### DISCUSSION TITLE
Amazing!!!

### DISCUSSION THREAD

**Discussion by moondisco** [djgreeb/Sampler-DJS-500#4] [General]
Nice work!!! I have Pioneer's DJS-1000. My biggest grip is that it can't be a slave only. When I have time I'll take dig into the board and dump it.

**Comment by mattpstv** [comment]
@moondisco i assume you've seen the latest DJS-1000 firmware update (v.1.2). drum machine mode/slave mode is pretty solid (only issue when in Drum Mode is that the BeatFX on a DJM don't use PROLINK to quantize FX applied to DJS-1000, though the non-PRO LINK beat detect usually does a decent enough job. i have two DJS-1000s and an sp-16 but I still really want to build a working DJS-500. I've been following this project for a few years since it first appeared on youtube and at the moment i'm waiting for a STM32F746G-DISCO board to arrive, but i look forward to the day i can give this thing a full workout. there's so many features on here that are no doubt born out of the mind of someone who actively uses a sampler in a DJ set, versus a traditional sampler designed by an engineering team with some input from DJs.

**Reply by moondisco** [reply]
I finally saw that update, and it's a nice touch. Definitely needed that, but there are other edits I would like to do. The biggest one is allowing two 1/4" inputs to be used separately, rather than only input 1/4" L as mono. I would like to have my 909 and 303 both sent to 1/4" 1L and 2R on separate pads in mono.

**Reply by mattpstv** [reply]
deja vu. i feel like i just had this convo in reddit. i feel you on the DJS-1000. i have a very long wish list of 'small' changes or features i wish the DJS had, so i feel you on the multiple independent ins. here's my unsolicited personal top 3: 1) a few shortcuts for quick copy/pasting track sequences, patterns, etc (opposed to having to navigate to screen manager, click SEQ, set the TRACK to all or current, click COPY, then manually switch patterns & finally paste); 2) FX2 like the sp-16 has (it's frustrating because there's even an open spot for fx2) or at least removing ducker from FX1 and adding ducking to each track independently; 3) ability to map Slices/Hot Slices via midi.
